Pinta.Resources/Icons.cs

```diff
@@ -238,6 +238,7 @@ public static class Icons
 	public const string ToolZoom = "tool-zoom-symbolic";
 
 	public const string ViewGrid = "view-grid";
+	public const string ViewMenu = "view-reveal-symbolic";
```

Comment:
The icon worked fine for me on macOS 👍  
For organization purposes, since this isn't a custom icon  it should go into `StandardIcons` and be named `ViewReveal` to match the underlying icon name

Reply:
Done!